sql/pgwire/v3.go

```diff
@@ -172,6 +172,7 @@ func (c *v3Conn) serve(authenticationHook func(string, bool) error) error {
 	for key, value := range map[string]string{
 		"client_encoding": "UTF8",
 		"DateStyle":       "ISO",
+		"server_version":  "9.1.0",
```

Comment:
Yeah, I think either use the latest at time of writing or the oldest version that has the properties we want (if the latter is difficult to determine then I'd just go with 9.5.0).